platform/commonUI/general/res/templates/controls/time-controller.html

```diff
@@ -35,13 +37,16 @@
 
         <span class="l-time-range-input" ng-controller="ToggleController as t2">
             <mct-control key="'datetime-field'"
-                         structure="{ format: parameters.format }"
-                         ng-model="ngModel.outer"
+                         structure="{ format: parameters.format, submit: updateBoundsFromForm }"
```

Comment:
Good catch - that was an intermediary change that should have been removed. (This is ultimately handled by `ng-blur` here.)